src/snowflake/snowpark/dataframe_reader.py

```diff
@@ -248,11 +264,12 @@ def csv(self, path: str) -> DataFrame:
         """Specify the path of the CSV file(s) to load.
 
         Args:
-            path: The stage location of a CSV file, or a stage location that has CSV files.
+            path: A path pointing to a local CSV file, a local directory that has CSV files, a CSV file on a stage, or a stage location that has CSV files.
```

Comment:
Given that this now has four possibilities, I'm wondering if we should write it like:

```
path: A path pointing to one of the following:
    - a local CSV file
    - a local directory that has CSV files
    - a CSV file on a stage
    - a stage location that has CSV files
```

cc @sfc-gh-mabrennan

Reply:
I like it either way so I'll let you decide.

Reply:
I think its ok to remain the same? Cause in the docs the length of this line looks just fine